test/java/build.gradle

```diff
@@ -6,22 +6,18 @@ plugins {
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
-// tag::repositories[]
 repositories {
     mavenLocal()
     mavenCentral()
     maven {
-        url 'https://jitpack.io'
+        url = uri('https://pkgs.dev.azure.com/hmcts/Artifacts/_packaging/hmcts-lib/maven/v1')
```

Comment:
I think this has ended up here from your other jira, I thought we were going to leave jitpack in and just add the azure repo tho.

Reply:
michael says this is going to be fine cos it only pulls in fortify so fine